src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -845,15 +845,75 @@ public FeedbackSessionResultsBundle getFeedbackSessionResultsForStudent(
 
     public String getFeedbackSessionResultsSummaryAsCsv(
             String feedbackSessionName, String courseId,
-            String userEmail, String filterText, boolean isMissingResponsesShown, boolean isStatsShown)
+            String userEmail, String filterText, boolean isMissingResponsesShown,
+            boolean isStatsShown, String questionNumber)
             throws EntityDoesNotExistException, ExceedingRangeException {
 
         return getFeedbackSessionResultsSummaryInSectionAsCsv(
-                feedbackSessionName, courseId, userEmail, null, filterText, isMissingResponsesShown, isStatsShown);
+                feedbackSessionName, courseId, userEmail, null, filterText,
+                isMissingResponsesShown, isStatsShown, questionNumber);
     }
 
     public String getFeedbackSessionResultsSummaryInSectionAsCsv(
             String feedbackSessionName, String courseId, String userEmail,
+            String section, String filterText, boolean isMissingResponsesShown, boolean isStatsShown, String questionNumber)
+            throws EntityDoesNotExistException, ExceedingRangeException {
+
+        long indicatedRange = section == null ? 2000 : -1;
+        FeedbackSessionResultsBundle results = getFeedbackSessionResultsForInstructorInSectionWithinRangeFromView(
```

Comment:
Good point. Now, I am reusing existing method `getFeedbackSessionResultsForInstructorFromQuestion` to retrieve results for one questions instead of all questions. But because I need to use `questionId` I end up being unable to test the fileContent of the downloaded file.